model_bump.go

```diff
@@ -342,3 +346,119 @@ func (bumps BUMPs) Value() (driver.Value, error) {
 
 	return string(marshal), nil
 }
+
+// MerklePathToBUMP transforms Merkle Path to BUMP
+func MerklePathToBUMP(merklePath *bc.MerklePath, blockHeight uint64) BUMP {
+	bump := BUMP{BlockHeight: blockHeight}
+
```

Comment:
What do you thing about creating helper/self descriptive functions inside this one?

At least maybe we can get the business logic related to creating BUMPPaths and use it in both cases:
MerklePathToBUMP + MerkleProofToBUMP